book/src/terminology.md

```diff
@@ -74,6 +74,10 @@ A globally unique identifier that is also a proof that the [entry](terminology.m
 
 The time, i.e. number of [slots](terminology.md#slot), for which a [leader schedule](terminology.md#leader-schedule) is valid.
 
+## execution key
+
+The key in the transaction that pays for the cost of including the transaction in the ledger.  This is the first key in the transaction.
```

Comment:
Worth suggesting here that an execution key is highly recommended to be set as Credit-Only?

Reply:
@mvines no, its CD to pay for the fee.  it just can't be used in any instruction.

Reply:
@mvines, I think it must be CD, otherwise the CO assumptions will fail.

Reply:
Yes my bad.